services/appnavstate/impl/src/main/kotlin/io/element/android/services/appnavstate/impl/DefaultAppNavigationStateService.kt

```diff
@@ -83,7 +82,7 @@ class DefaultAppNavigationStateService @Inject constructor(
         val currentValue = state.value.navigationState
         Timber.tag(loggerTag.value).d("Navigating to space $spaceId. Current state: $currentValue")
         val newValue: NavigationState.Space = when (currentValue) {
-            NavigationState.Root -> error("onNavigateToSession() must be called first")
+            NavigationState.Root -> return logError("onNavigateToSession()")
```

Comment:
Do not throw but just log and return early.
This could lead to crash because of node lifecycle `onDestroy` callback going from parent to children. 
See here for the appyx related question https://github.com/bumble-tech/appyx/issues/498